resources/dicts/patrols/new_cat.json

```diff
@@ -5,22 +5,45 @@
     "season": "Any",
     "tags": ["border", "new_cat_loner_warrior_litter1", "reputation", "meeting"],
     "intro_text": "Your patrol finds a loner who is interested in joining the Clan.",
+    "decline_text": "Your patrol decides not to confront the loner.",
     "success_text": [
-            "The patrol convinces the loner to join the Clan.",
-            "The patrol convinces the loner to join the Clan, and they join with their litter of kits."
+        "The patrol convinces the loner to join the Clan.",
+        "The patrol convinces the loner to join the Clan, and they join with their litter of kits."
     ],
     "fail_text": [
-            "The loner decides against joining after hearing more about Clan life."
+        "The loner decides against joining after hearing more about Clan life."
     ],
-    "decline_text": "Your patrol decides not to confront the loner.",
     "antagonize_text": "Your patrol drives the loner off of the territory.",
     "antagonize_fail_text": "The loner is unimpressed by their hostility and decides this Clan is not for them.",
     "chance_of_success": 40,
     "exp": 10,
     "win_skills": ["good speaker", "great speaker", "excellent speaker"],
-    "min_cats": 1,
+    "min_cats": 2,
     "max_cats": 6
 
+},
+{
+    "patrol_id": "gen_bord_newcatsolo1",
```

Comment:
in order to make sure this works for the patrol sprite, you should remove 'solo' from the ID and just increment the number